eng/scripts/Analyze-Code.ps1

```diff
@@ -20,44 +20,99 @@ Analyzing code with
 "@
 
 if ($CheckWasm) {
-  Invoke-LoggedCommand "rustup target add wasm32-unknown-unknown"
+  # Temporary fix to exit immediately on failure. LogError should Write-Error
+  # instead
+  $command = "rustup target add wasm32-unknown-unknown"
+  Invoke-LoggedCommand $command
+  if ($LastExitCode) {
+    Write-Error "Failed to execute $command"
```

Comment:
This works, but the `Invoke-LoggedCommand` already logs the command line so this becomes redundant. That said, you already did the work so I don't see any reason to block the PR either.